### PR TITLE
docs: uniform semicolon and quotation marks

### PR DESCRIPTION
**Pull request by YangFong**
#### Summary

- Use the semicolon if it is needed, and remove it if it is not.
- Single quotation marks are preferred for strings

#### Metadata

- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/map/keys/index.md [hunk 1]
```diff
@@ -35,9 +35,9 @@ A new {{jsxref("Map")}} iterator object.
 
 ```js
 const myMap = new Map();
-myMap.set("0", "foo");
-myMap.set(1, "bar");
-myMap.set({}, "baz");
+myMap.set('0', 'foo');
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/map/keys/index.md:38]: I personally prefer double quotes, but we haven't reached resolution about which side we want to unify on yet. In the meantime we would accept both, and if a page consistently uses one quote we won't change it.

File: files/en-us/web/javascript/reference/global_objects/map/size/index.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ myMap.set('a', 'alpha');
 myMap.set('b', 'beta');
 myMap.set('g', 'gamma');
 
-myMap.size // 3
+myMap.size; // 3
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/map/size/index.md:34]:
```suggestion
console.log(myMap.size); // 3
```

**Review by Josh-Cena — CHANGES_REQUESTED** [review]

**Comment by teoli2003** [comment]
We plan to use double quotes as they are the default for Prettier. The less configuration we need for Prettier, the better.

Note that I don't think it is a good use of our time to fix quotes to double-quotes, as this will be automated in one go once we use Prettier (hopefully very soon).

I'm going to close this PR.

**Comment by teoli2003** [comment]
(Feel free to open another PR with only the other changes, like using a `console.log()`.

**Comment by YangFong** [comment]
> We plan to use double quotes as they are the default for Prettier. The less configuration we need for Prettier, the better.
> 
> Note that I don't think it is a good use of our time to fix quotes to double-quotes, as this will be automated in one go once we use Prettier (hopefully very soon).
> 

Well, I'm glad to hear that.

---

I'll try adding 'console.log()'.

**Pull request closed without merging**